README.md

```diff
@@ -10,6 +10,10 @@ Selenium automation framework project, to prove key user journeys work.
 - Go here: https://chromedriver.chromium.org/downloads and download the latest chromedriver.
 - Put the chromedriver you download (and unzip) into your /usr/local directory
 
+- On MacOS:
+    - When you first run the UI tests on your machine, you might receive an error telling you that '“chromedriver” cannot be opened because the developer cannot be verified'. If you receive this error, open System Preferences > Security & Privacy > General and select 'Allow Anyway' alongside the messaege about chromdriver.
```

Comment:
Was going to leave it, but as I have added another comment: few spelling mistakes here.